source/_components/climate.modbus.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "Modbus"
+description: "Instructions how to integrate a Modbus thermostat within Home Assistant."
+date: 2018-01-29 9:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: modbus.png
+ha_category: Thermostat
+ha_release: pre 0.7
+ha_iot_class: "Local Polling"
+---
+
+
+The `modbus` thermostat allows you to use a sensor value (current temperature)
+and target value (target temperature) from [Modbus](http://www.modbus.org/)
+registers.
+
+To use your Modbus thermostat in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yml entry
+climate:
+  - platform: modbus
+    name: Watlow F4T
+    # Modbus TCP
+    slave: 1
+    # Control Loop 1 Setpoint
+    target_temp_register: 2782
+    # Universal Input 1 Module 1
+    current_temp_register: 27586
+    data_type: float
```

Comment:
Please don't add optional configuration parameters to the examples.
We'd like to keep examples as minimal as possible.